ansible/testbed.yaml

```diff
@@ -104,6 +104,7 @@
     - lab-a7260-01
   inv_name: lab
   auto_recover: 'True'
+  is_smartswitch: 'False'
```

Comment:
Can we make is_smartswitch to False by default and set true only for smart switch testbeds.

Reply:
This one I forgot to remove. 
False and field not present are the same. The check will take care of it according to the previous comment given by prince/congh